js/index.js

```diff
@@ -0,0 +1,113 @@
+const input = document.getElementById("user-input");
+//const searchButton = document.getElementById("search-button");
+const searchList = document.getElementById("search-list");
+const spinner = document.getElementById("spinner");
+
+const baseURL =
+  "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/";
+
+function debounce(func, interval) {
+  var timeout;
+  return function () {
+    var context = this,
+      args = arguments;
+    var later = function () {
+      timeout = null;
+      func.apply(context, args);
+    };
+    clearTimeout(timeout);
+    timeout = setTimeout(later, interval || 200);
+  };
+}
+
+function getSearchURL(input) {
```

Comment:
in this case, i see no reason to have a function just for concatenating (connecting) between 2 strings.